app/jobs/publish_feed_job.rb

```diff
@@ -4,6 +4,7 @@ class PublishFeedJob < ApplicationJob
   queue_as :feeder_publishing
 
   include PodcastsHelper
+  include PublishingNotify
```

Comment:
Redundant include?  (`PodcastHelper` also includes `PublishNotify`).

Reply:
yeah, that's true - I added it here first, then the helper later, but yeah, I can clean this up.

Reply:
fixed!